### PR TITLE
Update Golang and Kubernetes

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

- Update Golang to 1.13
- Update Kubernetes to 1.17.3

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle @hyy0322 @iawia002 

**Release note**:

```release-note
Update Golang to 1.13
Update Kubernetes to 1.17.3
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,59 +1,62 @@
...
-	k8s.io/gengo v0.0.0-20181106084056-51747d6e00da
-	k8s.io/kube-openapi v0.0.0-20181114233023-0317810137be // indirect
-	k8s.io/utils v0.0.0-20190607212802-c55fbcfc754a // indirect
-	sigs.k8s.io/yaml v1.1.0 // indirect
+	k8s.io/apimachinery v0.17.3
+	k8s.io/client-go v0.17.3
+	k8s.io/gengo v0.0.0-20190822140433-26a664648505
+)
+
+replace (
```
Comment (iawia002) [inline go.mod]: You don't need these because you don't use `k8s.io/kubernetes` as a package

File: go.mod [hunk 2]
```diff
@@ -1,59 +1,62 @@
...
-	golang.org/x/time v0.0.0-20181108054448-85acf8d2951c // indirect
-	golang.org/x/tools v0.0.0-20181106213628-e21233ffa6c3 // indirect
+	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
 	gopkg.in/go-playground/assert.v1 v1.2.1 // indirect
 	gopkg.in/go-playground/validator.v9 v9.23.0 // indirect
-	gopkg.in/inf.v0 v0.9.0 // indirect
 	gopkg.in/xanzy/go-gitlab.v0 v0.5.1
-	k8s.io/api v0.0.0-20190516230258-a675ac48af67
+	k8s.io/api v0.17.3
 	k8s.io/apiextensions-apiserver v0.0.0-20190516231611-bf6753f2aa24
```
Comment (iawia002) [inline go.mod]:
```suggestion
	k8s.io/apiextensions-apiserver v0.17.3
```

File: go.mod (hunk 1 above)
Comment (zhujian7) [reply go.mod]: OK

**Comment by zhujian7** [comment]
@iawia002 PTAL

**Pull request merged**